src/plugins/visualize/public/application/utils/use/use_saved_vis_instance.ts

```diff
@@ -44,7 +44,7 @@ export const useSavedVisInstance = (
     savedVisInstance?: SavedVisInstance;
     visEditorController?: IEditorController;
   }>({});
-  const visEditorRef = useRef<HTMLDivElement>(null);
+  const visEditorRef = useRef<HTMLDivElement | null>(null);
```

Comment:
nit: Docs note:
```
if you need the result of useRef to be directly mutable, include | null in the type of the generic argument.
```

That means, `visEditorRef.current` is mutable with such a type.
If you'll get rid of union type and leave only the `HTMLDivElement`, this will still save possible `null` value, but in that case  `visEditorRef.current` is a read-only property.

Reply:
Yeah, that's exactly why I added it here :) Since the file now behaves slightly different I need to make sure `visEditorRef` has a value in the tests, since otherwise it does not run through the initialization phase, so I need it to be a MutualRef object in that case. This seemed for me to be the easiest way.